articles/multifactor-authentication/developer/libraries/android/index.md

```diff
@@ -0,0 +1,150 @@
+---
+toc_title: Getting Started with Guardian for Android
+url: /multifactor-authentication/developer/libraries/android
+description: Installation, usage, and configuration options guide for Guardian for Android
+---
+
+# Guardian for Android
+The Guardian for Android Software Development Kit allows developers to create Android apps with Guardian functionality, providing easy and secure access to multifactor authentication with push notifications. For example, this toolkit gives you the power to build a 'white label' version of the Guardian application for your users, using your own look-and-feel.
+
+More information can be found on Guardian [here](/multifactor-authentication/guardian). For general multifactor discussion, read more [here](/multifactor-authentication).
+
+Get started using Guardian for Android below, or, if you're looking for a specific document, try the listing of [additional documents](#additional-documents) related to Guardian for Android.
+
+## Requirements
+
+Android API level 15+ is required in order to use the Guardian Android SDK.
+
+## Installation
+
+Guardian is available both in [Maven Central](http://search.maven.org) and [JCenter](https://bintray.com/bintray/jcenter). To start using *Guardian* add these lines to your `build.gradle` dependencies file:
+
+```gradle
+compile 'com.auth0.android:guardian-sdk:0.1.0'
+```
+
+_You can check for the latest version on the repository [Releases](https://github.com/auth0/GuardianSDK.Android/releases) tab, in [Maven](http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22guardian-sdk%22%20g%3A%22com.auth0.android%22), or in [JCenter](https://bintray.com/auth0/guardian-android)._
+
+After adding your Gradle dependency, make sure to remember to sync your project with Gradle files.
+
+## Dashboard Settings
+
+To enable Guardian Push Notifications for your users, go to the [Multifactor Auth](${manage_url}/#/guardian) section of the dashboard. Then toggle the **Push Notification** slider to enable it.
+
+![](/media/articles/mfa/guardian-dashboard.png)
+
+## Using the SDK
+
+
+`Guardian` is the core of the SDK. You'll need to create an instance of this class for your specific tenant/url.
+
+```java
+Uri url = Uri.parse("https://tenant.guardian.auth0.com/");
```

Comment:
we should use a variable for this. E.g. ${account.namespace} will give you back tenant.auth0.com (or the corresponding URL on the appliance).